packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -5,16 +5,17 @@ import { unstable_getMiniflareWorkerOptions } from 'wrangler';
 import { fileURLToPath } from 'node:url';
 import * as path from 'node:path';
 import { createCloudflareEnvironment } from './cloudflare-environment';
+import { getModuleFallbackHandler } from './module-fallback';
+import { getNodeCompatModules } from './node-compat';
 import type { FetchFunctionOptions } from 'vite/module-runner';
 import type { WorkerOptions } from 'miniflare';
 import type {
 	CloudflareEnvironmentOptions,
 	CloudflareDevEnvironment,
 } from './cloudflare-environment';
-import { getModuleFallbackHandler } from './module-fallback';
 import type { ResolveIdFunction } from './module-fallback';
 
-const wrapperPath = '__VITE_WRAPPER_PATH__';
+const wrapperPath = '__VITE_WORKER_ENTRY_POINT__';
```

Comment:
I changed this simply because it makes more sense to the user when they see an error that contains this in the stack trace.

Reply:
Sure. One thing I've come to realise is that the word 'entrypoint' is reserved for individual exports (e.g. named entrypoints). I've started using the word 'entry' for the files instead so this could be `__VITE_WORKER_ENTRY__`.